15-draft.json

```diff
@@ -1181,13 +1181,23 @@
       "items": {
         "type": "object",
         "properties": {
-          "url": {
-            "description": "The space's website according to their SpaceAPI endpoint",
+          "endpoint": {
+            "description": "The SpaceAPI endpoint of the space",
+            "type": "string"
+          },
+          "website": {
+            "description": "The website of the space",
             "type": "string"
           }
         },
-        "required": [
-          "url"
+        
+        "anyOf": [
+          "required": [
+            "endpoint"
+          ],
+          "required": [
+            "website"
+          ]
```

Comment:
@SpaceApi/core I made our discussed changes. I'm not 100% sure if `anyOf` works like this.